### DISCUSSION TITLE
Saving start up file?

### DISCUSSION THREAD

**Discussion by srjghseruikg** [mcmonkeyprojects/SwarmUI#427] [Q&A] [answered]
Is there a way i can save all my gen settings and have them show up each time i start the UI? Having to re-input everything is kind of a pain and i don't see a setting for it anywhere.

**Comment by mcmonkey4eva** [comment, marked as answer]
create a preset and title it `Default`

**Reply by srjghseruikg** [reply]
Ahhhh that's what the presets tab is for, okay. Thank you.

**Reply by Squirrelies** [reply]
> create a preset and title it `Default`

Suggestion to add this to documentation such as `Configuring default settings`.
